modules/migration-prerequisites.adoc

```diff
@@ -6,35 +6,51 @@
-* The {mtc-short} version must be the same on all clusters.
-ifdef::migrating-3-4[]
+
+.Direct image migration
+
+* You must ensure that the secure registry of the source cluster is exposed.
+
+.Direct volume migration
+
+* If your clusters use proxies, you must configure an Stunnel TCP proxy.
+
+ifdef::migrating-applications-3-4[]
+.Internal images
+
 * If your application uses internal images from the `openshift` namespace, you must ensure that the required versions of the images are present on the target cluster.
 +
 You can manually update an image stream tag in order to use a deprecated {product-title} 3 image on an {product-title} {product-version} cluster.
 endif::[]
-* Clusters:
-** The source cluster must be upgraded to the latest {mtc-short} z-stream release.
-** The cluster on which the `migration-controller` pod is running must have unrestricted network access to the other clusters.
-** The clusters must have unrestricted network access to each other.
-** The clusters must have unrestricted network access to the replication repository.
-** The clusters must be able to communicate using OpenShift routes on port 443.
-** The clusters must have no critical conditions.
-** The clusters must be in a ready state.
-
-* Volume migration:
-** The persistent volumes (PVs) must be valid.
-** The PVs must be bound to persistent volume claims.
-** If you copy the PVs by using the _move_ method, the clusters must have unrestricted network access to the remote volume.
-** If you copy the PVs by using the _snapshot_ copy method, the following prerequisites apply:
-*** The cloud provider must support snapshots.
-*** The volumes must have the same cloud provider.
-*** The volumes must be located in the same geographic region.
-*** The volumes must have the same storage class.
-
-* If you perform a direct volume migration in a proxy environment, you must configure an Stunnel TCP proxy.
-ifndef::migrating-applications-3-4[]
-* If you perform a direct image migration, you must expose the secure registry of the source cluster to external traffic.
+
+.Clusters
+
+* The source cluster must be upgraded to the latest {mtc-short} z-stream release.
+* The {mtc-short} version must be the same on all clusters.
+
+.Network
+
+* The clusters have unrestricted network access to each other and to the replication repository.
+* If you copy the persistent volumes with `move`, the clusters must have unrestricted network access to the remote volumes.
+ifdef::migrating-applications-3-4[]
+* You must enable the following ports on an {product-title} 3 cluster:
+** `8443` (API server)
+** `443` (routes)
+** `53` (DNS)
 endif::[]
+* You must enable the following ports on an {product-title} 4 cluster:
+** `6443` (API server)
+** `443` (routes)
+** `53` (DNS)
+* You must enable port `443` on the replication repository if you are using TLS.
+
+.Persistent volumes (PVs)
+
+* The PVs must be valid.
+* The PVs must be bound to persistent volume claims.
```

Comment:
> The PVs must be bound to persistent volume claims.
I don't think this is required.  @sergiordlr right?

Reply:
As far as I know this is required.